agent/state.go

```diff
@@ -18,7 +18,7 @@ const (
 	receivingAlgorithm
 	receivingData
 	running
-	resultsReady
+	results
```

Comment:
use a verb

Reply:
It's results so that the full event with status would be:  `results` `ready`. Rather than `resultsReady` `ready`

Reply:
state results is not really self explanatory here, a better term is required